extra_tests/snippets/stdlib_ssl_short_recv.py

```
@@ -0,0 +1,88 @@
+        server_context = ssl.SSLContext(ssl.PROTOCOL_TLS_SERVER)
+        server_context.load_cert_chain(CERTFILE)
+
+        sock, _ = listener.accept()
+        sock.settimeout(5.0)
+
+        ssock = server_context.wrap_socket(sock, server_side=True)
+        try:
+            ssock.sendall(DATA)
+        finally:
+            ssock.close()
+    except BaseException as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `BaseException`

(BLE001)